libraries/AP_HAL_ChibiOS/hwdef/ReaperF745/hwdef.dat

```diff
@@ -126,6 +126,7 @@ SPIDEV osd       SPI1 DEVID1 AT7456E_CS MODE0 10*MHZ 10*MHZ
 
 # no built-in compass and no external I2C so no compass
```

Comment:
So I guess this comment might be out of date now?

Reply:
I think just a copy and paste error